curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/657b1a03df3ec46eca276046.md

```diff
@@ -0,0 +1,48 @@
+---
+id: 657b1a03df3ec46eca276046
+title: Task 27
+challengeType: 22
+dashedName: task-27
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Anna: That's great to hear. Were the members of the team involved in any specific achievements or projects?
+Candidate2: Yes, our team was involved in a project to redesign the company's website.
+-->
+
+# --description--
+
+Listen to the dialogue and fill in the blanks with the correct words.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Anna: That's great to hear. _ the members of the team involved in any specific achievements or projects?`
+`Candidate2: Yes, our team _ involved in a project to _ the company's website.`
```

Comment:
```suggestion
`Yes, our team _ involved in a project to _ the company's website.`
```

Reply:
I'm adjusting some challenges to have just one character speaking for fill-in-the-blank.